tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -416,6 +416,10 @@ TEST(tfw_str_eq_cstr_pos, plain)
 	TfwStr *fox = make_plain_str(foxstr), *c, *end;
 	long i, offset = 0, foxlen = fox->len;
 
+	/* unlike to tfw_str_eq_cstr_off we don't  know which pointer we should use
+	 * to address string element at 'offset' position from begging of the string.
+	 * go though chunks, manually to get needed pointers
+	 */
```

Comment:
We raised question about complexity of this unit test in conference twice last week. May be it should  be commented?
About going over chunks of a plain string. Do we really need to make unit test  rely on implementation? I should have think about that before, but compound  and plain strings have the same interface, so unit tests are mostly the same for both. May be it is better to provide a single test function and pass compound or plain strings as parameter?